README.md

```diff
@@ -72,14 +82,28 @@ vm:
   org: misc
   size: 15G
   root_type: filesystem
-  nfs_ip: '192.168.10.52' #only this IP can access NFS on the storage server
+  filesystems:
+   - name: data
+     zfs_attributes:
+       quota: 50G
+     nfs_options:
+      - rw=@192.168.10.52
 interfaces:
   - mac: 'AA:BB:CC:FE:19:AA'
     ip:  '192.168.10.52'
   - mac: 'AA:BB:CC:FE:19:AB'
     ip:  '192.168.100.52'
 ```
 
+### Result
+
+Assuming the vm is named `web01`, these two filesystems will be created:
+
+|            Name            | ZFS attributes                                                                                   |
+|:--------------------------:|:-------------------------------------------------------------------------------------------------|
+| `tank/vms/misc/web01-root` | `quota=15G`, `reservation=15G`, `sharenfs=rw=@192.168.10.2,rw=@192.168.10.52,rw=@192.168.100.52` |
```

Comment:
no_root_squash?

Reply:
Forgot to update the example. It's fixed now.